### PR TITLE
Add docs for TagsInput

### PR DESCRIPTION
**Pull request by jessover9000**

### CODE REVIEW

File: packages/grafana-ui/src/components/TagsInput/TagsInput.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/TagsInput/TagsInput.mdx:1]: Adding `mdx` files on their own won't enable docs, they also need to be added to the story file. Like so: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/Button/Button.story.tsx#L13-L17 (also note the import at the top).

**Review by Clarity-89 — CHANGES_REQUESTED** [review]

File: packages/grafana-ui/src/components/TagsInput/TagsInput.mdx (hunk 1 above)
Comment (jessover9000) [reply packages/grafana-ui/src/components/TagsInput/TagsInput.mdx:1]: ohhh that's good to know, probably Tobias mentioned that half a year ago and I forgot it again :D Thanks!

Comment (Clarity-89) [reply packages/grafana-ui/src/components/TagsInput/TagsInput.mdx:1]: Yeah, I was wondering earlier why do we have a few `mdx` files not connected to stories :D

File: packages/grafana-ui/src/components/TagsInput/TagsInput.mdx [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+import { TagsInput } from './TagsInput';
+
+<Meta title="MDX|TagsInput" component={TagsInput} />
+
+# TagsInput
+
+A set of an input field and next to it that allows the user to add new tags. The added tags are previewed under the input and can be removed there by clicking the "X" icon. You can customize the width of the input.
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/TagsInput/TagsInput.mdx:8]:
```suggestion
A set of an input field and a button next to it that allows the user to add new tags. The added tags are previewed under the input and can be removed there by clicking the "X" icon. You can customize the width of the input.
```
Seems like a word is missing here, i suppose it's `button`?

**Review by Clarity-89 — APPROVED** [review]
Looks good, one minor comment.

**Pull request merged**